### DISCUSSION TITLE
중국어 폰트 적용 여부 질문드립니다!

### DISCUSSION THREAD

**Discussion by sol6521** [orioncactus/pretendard#122] [Q&A] [answered]
안녕하세요

앱에 Pretendard 폰트 적용 가능할지 테스트를 해보고 있는데,
현재 중국어 간체의 경우에는 Pretendard 폰트가 적용되지 않는다고 알고 있습니다.

그래서 중국어 간체에 대해 아이폰 System 폰트와 Pretendard 폰트 적용을 비교해보고 있는데요,
중국어 간체에도 Pretendard 폰트 적용이 됐는지 글자 간격이 System 폰트에 비해 조금 줄어들었다는 걸 확인할 수 있었습니다.
혹시 중국어 간체도 폰트 적용이 어느정도 되어있는건지 궁금합니다.

- 아래에는 비교하는 움짤 첨부할게요
![screenshot 2023-03-09_AM114750](https://user-images.githubusercontent.com/125871614/223903296-26346dbd-be63-4c91-b3be-6096c81d8db0.gif)

**Comment by orioncactus** [comment, marked as answer]
안녕하세요! Pretendard에 관심을 가져주셔서 감사드립니다 :D

스크린샷으로 보여주신 `长门鱼马` 문자는 Pretendard와 일본 환경에 맞는 한자가 포함된 Pretendard JP에 모두 포함되어 있지 않습니다. 표시되는 한자는 PingFang SC에 있는 문자로 보여지는데요, Pretendard를 적용했을 때 한자 자간이 바뀌는 모습은 의아하네요 🤔 혹시 한자가 포함되어 있지 않은 다른 한글 글꼴을 적용하면 한자 자간이 변하지 않는지 궁금합니다.

**Reply by sol6521** [reply]
네이버 나눔고딕 네오로 똑같은 글자 확인해봤는데,
![screenshot 2023-03-09_PM070342](https://user-images.githubusercontent.com/125871614/224203230-41343a91-88c8-42c5-a00c-2985dfcb17a3.gif)
자간이 변경되긴 하더라구요,,ㅎㅎ

**Reply by orioncactus** [reply]
다른 글꼴로도 확인해주시고 새로운 정보를 알려주셔서 감사드립니다! 다른 글꼴에서도 동일하게 발생하는 것을 보면 시스템에서 따로 처리를 하는 것 같습니다 👀 참고로 Pretendard에서는 해당 문자와 한자 사이에서 따로 처리하는 기능이 없습니다 😮
